### PR TITLE
feat: Updates for Inferentia-Trainium workshop

### PR DESCRIPTION
**Pull request by sanjeevrg89**
### What does this PR do?

1. Added a variable for kuberay operator (true or false); default is set to true to have no impact on the current module
2. Updated variables.tf and eks.tf for to accomodate the changes for inferentia workshop

🛑 Please open an issue first to discuss any significant work and flesh out details/direction - we would hate for your time to be wasted.
Consult the [CONTRIBUTING](https://github.com/awslabs/data-on-eks/blob/main/CONTRIBUTING.md#contributing-via-pull-requests) guide for submitting pull-requests.

### More

- [ yes] Yes, I have tested the PR using my local account setup (Provide any test evidence report under Additional Notes)
- [ yes] Mandatory for new blueprints. Yes, I have added a example to support my blueprint PR
- [ yes] Mandatory for new blueprints. Yes, I have updated the `website/docs` or `website/blog` section for this feature
- [ yes] Yes, I ran `pre-commit run -a` with this PR. Link for installing [pre-commit](https://pre-commit.com/) locally

### For Moderators

- [ ] E2E Test successfully complete before merge?

### CODE REVIEW

File: ai-ml/trainium-inferentia/variables.tf [hunk 1]
```diff
@@ -116,3 +116,37 @@ variable "inf2_48xl_desired_size" {
...
+  #     username = "role1"
+  #     groups   = ["system:masters"]
+  #   }
+}
+
+variable "kms_key_admin_roles" {
+  description = "list of role ARNs to add to the KMS policy"
+  type        = list(string)
+  default     = []
+
```
Comment (vara-bonthu) [inline ai-ml/trainium-inferentia/variables.tf:151]: remove space and run `pre-commit run -a ` locally to format

File: ai-ml/trainium-inferentia/addons.tf [hunk 2]
```diff
@@ -267,7 +267,7 @@ module "eks_data_addons" {
   #---------------------------------------
   # Kuberay Operator
   #---------------------------------------
-  enable_kuberay_operator = true
+  enable_kuberay_operator = var.enable_kuberay_operator
```
Comment (vara-bonthu) [inline ai-ml/trainium-inferentia/addons.tf:270]: Are you disabling Kuberay operator as a part of the workshop?

File: ai-ml/trainium-inferentia/variables.tf [hunk 3]
```diff
@@ -116,3 +116,37 @@ variable "inf2_48xl_desired_size" {
   type        = number
   default     = 0
 }
+
+variable "enable_kuberay_operator" {
+  description = "Flag to enable kuberay operator"
+  type        = bool
+  default     = true
```
Comment (vara-bonthu) [inline ai-ml/trainium-inferentia/variables.tf:123]: `true` is fine as false may break the existing blueprints

**Review by askulkarni2 — APPROVED** [review]

File: ai-ml/trainium-inferentia/addons.tf (hunk 2 above)
Comment (sanjeevrg89) [reply ai-ml/trainium-inferentia/addons.tf:270]: yes @vara-bonthu

**Pull request merged**